params.yaml

```diff
@@ -0,0 +1,41 @@
+graph-defaults:
+  levels: 4
+  hierarchical: true
+  name: hierarchical
+
+baseline-defaults: &baseline-defaults  # defaults across training and evaluation stage
+  num_workers: 8 
+  batch_size: 1
+  hidden_dim: 128
+  processor_layers: 4
+  model: hi_lam
+  graph: hierarchical
+  epochs: 2
+  ar_steps_train: 1
+  lr: 0.001
+  val_interval: 1
+  ar_steps_eval: 4
+  val_steps_to_log:
+    - 1
+    - 2
+    - 4
+  metrics_watch: val_rmse, mse, random_metric
+  num_nodes: &num_nodes 2
+
+baseline-eval:
+  <<: *baseline-defaults
+  num_nodes: 1
+
+slurm-defaults: &slurm-defaults
```

Comment:
could you add a comment here to say how these are passed to SLURM? I see `${slurm_defaults}` used in `dvc.yaml`, but it would be good to explain here why and how this works (so that people don't have to try and dig through the dvc documentation to work that out)